src/content/es/2019/media.md

```diff
@@ -0,0 +1,413 @@
+
+<figure>
+  <a href="/static/images/2019/media/fig17_usage_of_accept-ch_http_v_html.png">
+    <img src="/static/images/2019/media/fig17_usage_of_accept-ch_http_v_html.png" alt="Figura 17. Uso del Accept-CH: HTTP versus HTML." aria-labelledby="fig17-caption" aria-describedby="fig17-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=284657706&format=interactive">
+  </a>
+  <div id="fig17-description" class="visually-hidden">Gráfico de barras mostrando que un 71% usa el 'meta http-equiv', un 30% usa el encabezado 'Accept-CH' HTTP y un 1% usa ambos.</div>
+  <figcaption id="fig17-caption">Figura 17. Uso del encabezado <code>Accept-CH</code> versus la etiqueta <code>&lt;meta></code> equivalente.</figcaption>
+</figure>
+
+El uso de la etiqueta `<meta>` en HTML para invocar los <i lang="en">Client Hints</i> es bastante más común en comparación con el del encabezado HTTP. Esto es un claro reflejo de la comodidad de modificar el marcado de las plantillas en comparación con añadir los encabezados HTTP en cajas intermedias. De todos modos, analizando el uso del encabezado HTTP, más del 50% de estos casos provienen de una sola plataforma SaaS (Mercado).
+
+De los <i lang="en">Client Hints</i> solicitados, la mayoría de las páginas los usan para los tres casos originales de `DPR`, `ViewportWidth` y `Width`. Por supuesto, el <i lang="en">Client Hint</i> `Width` necesita del uso de `<img sizes>` para que el navegador tenga el contexto necesario relativo al diseño.
+
+<figure>
+  <a href="/static/images/2019/media/fig18_enabled_client_hints.png">
+    <img src="/static/images/2019/media/fig18_enabled_client_hints.png" alt="Figura 18. Client Hints habilitados." aria-labelledby="fig18-caption" aria-describedby="fig18-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=1878506264&format=interactive">
+  </a>
+  <div id="fig18-description" class="visually-hidden">Un gráfico circular de estilo donut que muestra que el 26,1% de los <i lang="en">client hints</i> usa 'dpr', 24,3% usa 'viewport-width', 19,7% usa 'width', 6,7% usa 'save-data', 6,1% usan 'device-memory', 6,0% usan 'downlink', 5,6% usan 'rtt' y 5,6% usan 'ect'</div>
+  <figcaption id="fig18-caption">Figura 18. <i lang="en">Client Hints</i> habilitados.</figcaption>
+</figure>
+
+Los <i lang="en">Client Hints</i> relacionados con la red, `downlink`, `rtt`, y `ect`, solamente están disponibles en Android Chrome.
+
+### <i lang="en">Lazy loading</i>
```

Comment:
We generally don't put the `<i lang="en"></i>` elements in headings as it messes up the automatically created short links. See discussion here: #656 

```suggestion
### Lazy loading
```